src/core/geometry/qgsgeometry.cpp

```diff
@@ -80,9 +80,8 @@ QgsGeometry::QgsGeometry( std::unique_ptr<QgsAbstractGeometry> geom )
   d->ref = QAtomicInt( 1 );
 }
 
-QgsGeometry::QgsGeometry( const QgsGeometry &other )
+QgsGeometry::QgsGeometry( const QgsGeometry &other ): d( other.d )
```

Comment:
Pedantic, but qgis convention is to put the : on a new line

Reply:
done